src/test/java/com/adyen/ClientTest.java

```diff
@@ -56,34 +54,6 @@ public void testConfigLiveNoPrefixCheckoutClient() {
     @Test
     public void testClientCertificateAuth() {
         Client client = new Client(trustStore, clientKeyStore, clientKeyStorePassword, apiKey, null);
-
         Assert.assertEquals(Environment.LIVE, client.getConfig().getEnvironment());
-        Assert.assertEquals("https://checkoutcert-live-eu.adyen.com/checkout", client.getConfig().getCheckoutEndpoint());
-        assertCommonEndpoints(client.getConfig());
-    }
-
-    @Test
-    public void testClientCertificateAuth_AU() {
-        Client client = new Client(trustStore, clientKeyStore, clientKeyStorePassword, apiKey, Region.AU);
-
-        Assert.assertEquals(Environment.LIVE, client.getConfig().getEnvironment());
-        Assert.assertEquals("https://checkoutcert-live-au.adyen.com/checkout", client.getConfig().getCheckoutEndpoint());
```

Comment:
I would be nice to find another way to test the `Region` functionality still works.